ui/app/components/ui/icon/icon.component.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Icon = ({ className, width, height, color, type }) => {
+  const svgs = {
+    send: (
+      <svg className={className} width={width} height={height} viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why did you put the svgs here instead of in `.svg` files in the `images/` directory?

Reply:
my thinking is that it's faster to render the svgs directly inline (it's basically just template data anyway). this means that there's no http request for svgs as image assets (faster renders since we're removing that overhead)

however, I realise the downside of this is that updating these assets becomes a bit more complicated (though, the "copy as svg text" functionality in figma makes it only slightly more so?)

Reply:
additionally this lets us modify the svg's fill prop easily to change an SVG colour... otherwise you'd need multiple assets for each colour treatment